apps/src/javalab/Playground.js

```diff
@@ -31,6 +31,11 @@ export default class Playground {
     );
   }
 
+  onStop() {
+    const audioElement = this.getAudioElement();
+    this.resetMediaElement(audioElement);
```

Comment:
nit: just call `this.resetAudioElement()` here?

Reply:
Ah good call, must have missed this on refactor :)